contracts/standard/permission/ArbitrablePermissionList.sol

```diff
@@ -51,14 +50,14 @@ contract ArbitrablePermissionList is PermissionInterface, Arbitrable {
      *  @param challenger Address of the challenger, if any.
      *  @param value The value of the item.
      *  @param newStatus The new status of the item.
-     *  @param newDisputed The new disputed state of the item.
+     *  @param disputed The item is being disputed.
      */
     event ItemStatusChange(
         address indexed submitter,
         address indexed challenger,
         bytes32 indexed value,
         ItemStatus newStatus,
-        bool newDisputed
+        bool disputed
```

Comment:
Doesn't seem like there is any logic around it being newly disputed. It always just broadcasts whether the item is currently disputed.

Reply:
It is the new value of disputed. It does change. E.g. `challengeRegisterItem`.

Reply:
Now that I look at it `newStatus` should just be `status` too in my opinion. It is unnecessary to say new in front of the variable name just because it changed.

Reply:
`status` and `disputed` sounds good.